haystack/tracing/opentelemetry.py

```diff
@@ -0,0 +1,45 @@
+import contextlib
+from typing import Optional, Dict, Any, Iterator
+
+from haystack.lazy_imports import LazyImport
+from haystack.tracing import Tracer, Span
+from haystack.tracing import utils as tracing_utils
+
+
+with LazyImport("Run 'pip install opentelemetry-sdk'") as opentelemetry_import:
+    import opentelemetry
+    import opentelemetry.trace
```

Comment:
is that fine or do you explicitly want to install it?

Reply:
I think it's fine, you already have to take action in order to use Otel, we'll add specific instructions to the docs that you also have to install the package.